src/theme/ItaliaTheme/Blocks/_ribbonCardTemplate.scss

```diff
@@ -8,11 +8,11 @@
     .etichetta {
       position: absolute;
       top: 3rem;
-      right: 0;
-      left: 0;
-      width: 100%;
+      right: 0px !important;
+      left: auto;
+      width: 75%;
       justify-content: inherit;
-      padding: 0 3rem;
+      padding-right: 2rem;
```

Comment:
Guardando quello che è l'html e le classi associate in bootstrap-italia, probabilmente fare una cosa di questo tipo"
```
.ribbon-card-template {
  .card,
  .card.card-big {
    .dates {
      font-size: 0.8em;
      text-transform: uppercase;
    }

    .etichetta {
      position: absolute;
      max-width: calc(100% - 176px);
      justify-content: inherit;

```
 Il calcolo sulla width è basato sui valori di margine del ribbon e la sua larghezza (48px e 32 px), il posizionamento assoluto a sinistra (48px, che viene mantenuto dalla classe `.card`, e un ulteriore spaziatura aggiunta per simmetria con il ribbon (48px come il suo margine a sx).
Questo perché abbiamo visto come alle card e alle linee guida piaccia cambiare, in questo modo mantieni le spaziature date da bootstrap-italia, che se cambiano cambiano tutte assieme a bump di versione.
Se vuoi definire delle variabili puoi usare queste
```
$ribbon-spacing-h: calc($spacer * 3);
$ribbon-width: calc($spacer * 2);

```
che usano la variabile e i moltiplicatori definiti da bootstrap-italia come qui https://italia.github.io/bootstrap-italia/docs/organizzare-gli-spazi/spaziature.
La somma finale quindi sarebbe 
```
 max-width: calc(100% - $ribbon-spacing-h * 3 - $ribbon-width);
```
Possono esser definite colocate qui, oppure messe e importate dalle site-variables